packages/util/src/create-universal-portal.ts

```diff
@@ -8,7 +8,7 @@ export function createUniversalPortal(
   container: Element,
   key?: null | string
 ): ReactPortal {
-  if (!canUseDOM) return null
+  if (!canUseDOM()) return null
```

Comment:
:)   nice catch.  Yeah, I've seen this at least one other time where exenv's API was a bool and this is a () => bool